plugins/make_plugin_rpm

```diff
@@ -169,7 +169,7 @@ sed -i "s/__ARCH__/${arch}/g" SPECS/plugin.spec
 sed -i "s/__PACKAGE_NAME__/${package_name}/g" SPECS/plugin.spec
 sed -i "s|__INSTALL_DIR__|${installs}|g" SPECS/plugin.spec
 sed -i "s/__REQUIRES__/${requirements}/g" SPECS/plugin.spec
-sed -i "s/Requires: foglamp/Requires: foglamp (${foglamp_version})/g" SPECS/plugin.spec
+sed -i "s/Requires: foglamp/Requires: foglamp(${foglamp_version})/g" SPECS/plugin.spec
```

Comment:
Remove this line and refer  https://github.com/foglamp/foglamp-pkg/pull/19/commits/d9db7f7f8c3564a223cbaf3c9b7826c472f92128#r282043778